src/compas_timber/connections/l_miter.py

```diff
@@ -41,13 +41,13 @@ class LMiterJoint(Joint):
     @property
     def __data__(self):
         data = super(LMiterJoint, self).__data__
-        data["beam_a"] = self.beam_a_guid
-        data["beam_b"] = self.beam_b_guid
+        data["beam_a"] = self.beam_a
```

Comment:
it is on purpose that guids are serialized here instead of the beams themselves. 
the beam objects are already serialized as part of the `Model` serialization. if these are assigned to `__data__` the same beam object might be serialized multiple times resulting in bloated files.